src/ripple/app/paths/RippleCalc.cpp

```diff
@@ -100,6 +100,12 @@ RippleCalc::Output RippleCalc::rippleCalculate (
         flowV1Out.setResult (result);
         flowV1Out.actualAmountIn = rc.actualAmountIn_;
         flowV1Out.actualAmountOut = rc.actualAmountOut_;
+        if (result != tesSUCCESS && !rc.permanentlyUnfundedOffers_.empty ())
+        {
+            flowV1Out.removableOffers.reserve (
```

Comment:
This is duplicate comment that, I think, github considered to be outdated.  I'm just trying to make sure the comment is not lost.

I suspect the `reserve` is not necessary since it is immediately followed by a `move`.